main.py

```diff
@@ -18,23 +27,51 @@
     help="Take temperature in Celsius and convert to Fahrenheit"
 )
 
+# parser.add_argument(
+#     "temperature", type=float,
+#     help="temperature", action="store", nargs='?'
+# )
+#
+# scale_group = parser.add_mutually_exclusive_group(required=False)
+#
+# scale_group.add_argument(
+#     "-f", "--fahrenheit", action="store_true",
+#     help="use flag -f(--fahrenheit) and provide one value for converting"
+# )
+#
+# scale_group.add_argument(
+#     "-c", "--celsius", action="store_true",
+#     help="use flag -c(--celsius) and provide one value for converting"
+# )
+
 args = parser.parse_args()
+# print(sys.argv)
```

Comment:
@MariaMaximova, I suggest using `--verbose` argument for print it: if the argument is present, then you can print this debug info.

Reply:
@MariaMaximova, you can use logging package for logging (do not forget to setup logging level)